src/web/web-service/frontend/res/css/styles.css

```diff
@@ -14,231 +14,416 @@
-  max-height: 300px;
+
+_:-ms-input-placeholder,
+ :root .demo-graph {
+    max-height: 300px;
 }
+
+
 /* TODO end */
+
 .demo-graph:nth-child(1) {
-  color: #00b9d8;
+    color: #00b9d8;
 }
+
 .demo-graph:nth-child(2) {
-  color: #d9006e;
+    color: #d9006e;
 }
 
 .demo-cards {
-  align-items: flex-start;
-  align-content: flex-start;
+    align-items: flex-start;
+    align-content: flex-start;
 }
+
 .demo-cards .demo-separator {
-  height: 32px;
+    height: 32px;
 }
+
 .demo-cards .mdl-card__title.mdl-card__title {
-  color: white;
-  font-size: 24px;
-  font-weight: 400;
+    color: white;
+    font-size: 24px;
+    font-weight: 400;
 }
+
 .demo-cards ul {
-  padding: 0;
+    padding: 0;
 }
+
 .demo-cards h3 {
-  font-size: 1em;
+    font-size: 1em;
 }
+
 .demo-updates .mdl-card__title {
-  min-height: 200px;
-  background-image: url('images/dog.png');
-  background-position: 90% 100%;
-  background-repeat: no-repeat;
+    min-height: 200px;
+    background-image: url('images/dog.png');
+    background-position: 90% 100%;
+    background-repeat: no-repeat;
 }
+
 .demo-cards .mdl-card__actions a {
-  color: #00BCD4;
-  text-decoration: none;
+    color: #00BCD4;
+    text-decoration: none;
 }
 
 .demo-options h3 {
-  margin: 0;
+    margin: 0;
 }
+
 .demo-options .mdl-checkbox__box-outline {
-  border-color: rgba(255, 255, 255, 0.89);
+    border-color: rgba(255, 255, 255, 0.89);
 }
+
 .demo-options ul {
-  margin: 0;
-  list-style-type: none;
+    margin: 0;
+    list-style-type: none;
 }
+
 .demo-options li {
-  margin: 4px 0;
+    margin: 4px 0;
 }
+
 .demo-options .material-icons {
-  color: rgba(255, 255, 255, 0.89);
+    color: rgba(255, 255, 255, 0.89);
 }
+
 .demo-options .mdl-card__actions {
-  height: 64px;
-  display: flex;
-  box-sizing: border-box;
-  align-items: center;
+    height: 64px;
+    display: flex;
+    box-sizing: border-box;
+    align-items: center;
 }
 
-
-
 #loader {
-  position: absolute;
-  left: 50%;
-  top: 300px;
-  z-index: 1;
-  width: 75px;
-  height: 75px;
-  margin: -75px 0 0 -75px;
-  border: 16px solid #13f3f3;
-  border-radius: 50%;
-  border-top: 16px solid #3498db;
-  width: 75px;
-  height: 75px;
-  -webkit-animation: spin 2s linear infinite;
-  animation: spin 2s linear infinite;
+    position: absolute;
+    left: 50%;
+    top: 300px;
+    z-index: 1;
+    width: 75px;
+    height: 75px;
+    margin: -75px 0 0 -75px;
+    border: 16px solid #13f3f3;
+    border-radius: 50%;
+    border-top: 16px solid #3498db;
+    width: 75px;
+    height: 75px;
+    -webkit-animation: spin 2s linear infinite;
+    animation: spin 2s linear infinite;
 }
 
 @-webkit-keyframes spin {
-  0% { -webkit-transform: rotate(0deg); }
-  100% { -webkit-transform: rotate(360deg); }
+    0% {
+        -webkit-transform: rotate(0deg);
+    }
+    100% {
+        -webkit-transform: rotate(360deg);
+    }
 }
 
 @keyframes spin {
-  0% { transform: rotate(0deg); }
-  100% { transform: rotate(360deg); }
+    0% {
```

Comment:
Don't change previous format, even it is bad/wrong. We may change it in separate PR, but it is out of the scope of our project and this PR.